sdk/storage/azure_storage_blob/src/lib.rs

```diff
@@ -3,13 +3,17 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 // Code generated by Microsoft (R) Rust Code Generator. DO NOT EDIT.
 
+mod clients;
 mod generated;
 
-pub use crate::generated::clients::*;
+pub(crate) mod pipeline;
+
+pub use clients::*;
 
-pub mod models {
-    pub use crate::generated::enums::*;
-    pub use crate::generated::models::*;
-}
+pub mod models;
+
+pub use crate::generated::clients::*;
+pub(crate) use blob_client::BlobClient as GeneratedBlobClient;
 
-pub use blob_client::BlobClient;
+pub use crate::generated::clients::blob_blob_client::BlobBlobClientGetPropertiesOptions;
+pub use crate::generated::clients::blob_client::BlobClientOptions;
```

Comment:
Why export them from the root for the crate? Just have the modules within the crate import `crate::generated::clients::BlobClient`. Re-exporting types only for the crate like this is going to lead to a lot of confusion, I think. It's not typical.

Reply:
So for L16 `pub(crate) use blob_client::BlobClient as GeneratedBlobClient;`, I can change this to in-line importing:
`generated::clients::blob_client::BlobClient as GeneratedBlobClient`

wrt L18-19, I re-exported them because I thought we wanted all client options accessible from the root ([guidelines section](https://azure.github.io/azure-sdk/rust_introduction.html#rust-client-configuration-namespace)). Without re-exporting them, we would have to import them as:
```rust
use crate::generated::{BlobClientOptions, BlobBlobClientGetPropertiesOptions};
```

Reply:
Gotcha. I took a note of this and will apply this feedback next PR-- but will not apply any changes as the structure of the generated imports completely changed with the newly generated versions.